tutorials/automate-tasks-using-cron/index.mdx

```diff
@@ -0,0 +1,186 @@
+| `@daily`    	| Run once a day, `0 0 * * *`.                                                                                     	|
+| `@midnight` 	| (same as `@daily`)                                                                                               	|
+| `@hourly`   	| Run once an hour, `0 * * * *`.                                                                                   	|
+
+## Managing crontabs
+
+Once you have decided when you want to run your cronjobs, privde the info in your `crontab` file so that the daemon can read it. 
+
+<Message type="important">
+
+The `crontab` file is a regular text file, however it is not recommended to edit it directly. Use the `crontab` command instead, which will also check your edits for syntax errors. 
+
+</Message>
+
+1. Open the `crontab` file: 
+    ```
+    crontab -e 
+    ```
+
+    If you are running the command for the first time, and no crontab exists for your user, a new one will be created. The following prompt displays:
+
+    ```
+    Select an editor.  To change later, run 'select-editor'.
+    1. /bin/nano        <---- easiest
+    2. /usr/bin/vim.basic
+    3. /usr/bin/vim.tiny
+    4. /bin/ed
+
+    Choose 1-4 [1]:
+    ```
+
+    Enter the number corresponding to editor of your choice, alternatively just press `Enter` to use the default choice, `nano` - which is the easiest to use option. 
+
+2. Edit your crontab by adding your tasks at the end of the file. 
+
+    <Message type ="note"> 
+
+    It contains by default some commented-out information how to edit the crontab file. Go to the end of the file to add your tasks. 
+
+    </Message>
+
+    ```
+    # Edit this file to introduce tasks to be run by cron.
+    #
+    # Each task to run has to be defined through a single line
+    # indicating with different fields when the task will be run
+    # and what command to run for the task
+    #
+    # To define the time you can provide concrete values for
+    # minute (m), hour (h), day of month (dom), month (mon),
+    # and day of week (dow) or use '*' in these fields (for 'any').
+    #
+    # Notice that tasks will be started based on the cron's system
+    # daemon's notion of time and timezones.
+    #
+    # Output of the crontab jobs (including errors) is sent through
+    # email to the user the crontab file belongs to (unless redirected).
+    #
+    # For example, you can run a backup of all your user accounts
+    # at 5 a.m every week with:
+    # 0 5 * * 1 tar -zcf /var/backups/home.tgz /home/
+    #
+    # For more information see the manual pages of crontab(5) and cron(8)
+    #
+    # m h  dom mon dow   command
+    ```
+
+    Once done, press `CTRL` and `O` to save your modifications, then  `CTRL` and `X` to quit the text editor. 
+
+<Message type="tip">
+
+`crontab` will rembmer your choice of editor. When you run `crontab -e` in the future, it will open the file automatically in the same text editor. 
```

Comment:
```suggestion
`crontab` will remember your choice of editor. When you run `crontab -e` in the future, it will open the file automatically in the same text editor. 
```